mods/ra/maps/sarin-gas-3-controlled-burn/controlledburn.lua

```diff
@@ -99,6 +107,28 @@ PowerDownTeslas = function()
 	end
 end
 
+PrepareFinishingHunt = function(player)
+	local buildings = GetBaseBuildings(player)
+
+	Trigger.OnAllKilledOrCaptured(buildings, function()
+		Utils.Do(player.GetGroundAttackers(), IdleHunt)
```

Comment:
```suggestion
		Utils.Do(player.GetGroundAttackers(), function(actor)
			actor.Hunt()
		end)
```

then it should look like this

Reply:
Done.